app/models/meter_reading.rb

```diff
@@ -0,0 +1,29 @@
+class MeterReading < ApplicationRecord
+  VALID_DATE = /([12]\d{3}-(0[1-9]|1[0-2])-(0[1-9]|[12]\d|3[01]))/.freeze
+
+  belongs_to :billing_contract, optional: true
+
+  validates :date, presence: true, format: { with: VALID_DATE,
+                                             message: 'is invalid' }
+  validates :value, presence: true, numericality: { only_integer: true }
```

Comment:
should have validation that it can't be smaller than previous one

Reply:
done